## STAR METHODS

### RESOURCE AVAILABILITY

#### Lead contact

Requests for access to OpenPedCan raw data and/or specimens may be directed to, and will be fulfilled by Jo Lynne Rokita (rokita@chop.edu).

#### Materials availability


#### Data and code availability


### METHOD DETAILS

#### Biospecimen Collection


#### Nucleic acids extraction and library preparation


#### Data generation


#### DNA WGS Alignment


#### Quality Control of Sequencing Data


#### Germline Variant Calling

##### SNP calling for B-allele Frequency (BAF) generation




#### Somatic Mutation Calling


##### SNV and indel calling


##### VCF annotation and MAF creation


##### Gather SNV and INDEL Hotspots


##### Consensus SNV Calling


#### Somatic Copy Number Variant Calling (WGS samples only)


##### Consensus CNV Calling


#### Somatic Structural Variant Calling (WGS samples only)


#### Gene Expression


##### Abundance Estimation


##### Gene Expression Matrices with Unique HUGO Symbols


##### Gene fusion detection


### QUANTIFICATION AND STATISTICAL ANALYSIS



##### Focal Copy Number Calling (`focal-cn-file-preparation` analysis module)



##### Gene Set Variation Analysis (`gene-set-enrichment-analysis` analysis module)




##### Fusion prioritization (`fusion_filtering` analysis module)


#### Oncoprint figure generation (`oncoprint-landscape` analysis module)


#### Mutational Signatures (`mutational-signatures` analysis module)



### Tumor Mutation Burden (`snv-callers` analysis module)


#### Clinical Data Harmonization

##### WHO Classification of Disease Types


##### Molecular Subtyping


#### TP53 Alteration Annotation (`tp53_nf1_score` analysis module)


#### Prediction of participants' genetic sex


#### Selection of independent samples (`independent-samples` analysis module)


#### Quantification of Telomerase Activity using Gene Expression Data (`telomerase-activity-prediction` analysis module)


### Survival models (`survival-analysis` analysis module)


### KEY RESOURCES TABLE


